Diary/Resource/Common/DiaryItemManager.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  DiaryItemManager.swift
+//  Diary
+//
+//  Created by SummerCat and som on 2022/12/28.
+//
+
+import Foundation
+
+final class DiaryItemManager {
+    static let shared = DiaryItemManager()
+    private var diaryItem: DiaryModel?
+    
+    func saveDiaryWith(title: String, body: String) {
```

Comment:
```suggestion
    func saveDiary(with title: String, body: String) {
```

with을 메소드이름에 포함하지말고 매개변수 레이블로 빼는게 조금 더 swift 스럽다고 생각합니다.

Reply:
![image](https://user-images.githubusercontent.com/98260324/210293637-c5030568-eb62-4d3d-a8e3-df15f03532c4.png)

`body`와 `title`이 하나의 일기를 이루는 요소이기 때문에 represent parts of a single abstraction인 경우에 해당된다고 생각해서 `with`를 메서드 이름으로 빼주었습니다.